docs/docs/cmd/spo/group/group-member-add.mdx

```diff
@@ -48,54 +48,42 @@ For the `userIds`, `userNames`, `emails`, `aadGroupIds` or `aadGroupNames` optio
 
 ## Examples
 
-Add a user with the userNames parameter to a SharePoint group with the groupId parameter
+Add a user with the userNames parameter to a SharePoint group with the groupId parameter.
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userNames "Alex.Wilber@contoso.com"
 ```
 
-Add multiple users with the userNames parameter to a SharePoint group with the groupId parameter
+Add multiple users with the userNames parameter to a SharePoint group with the groupId parameter.
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userNames "Alex.Wilber@contoso.com, Adele.Vance@contoso.com"
 ```
 
-Add a user with the emails parameter to a SharePoint group with the groupName parameter
+Add a user with the emails parameter to a SharePoint group with the groupName parameter.
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupName "Contoso Site Owners" --emails "Alex.Wilber@contoso.com"
 ```
 
-Add multiple users with the emails parameter to a SharePoint group with the groupName parameter
+Add multiple users with the emails parameter to a SharePoint group with the groupName parameter.
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupName "Contoso Site Owners" --emails "Alex.Wilber@contoso.com, Adele.Vance@contoso.com"
 ```
 
-Add a user with the userIds parameter to a SharePoint group with the groupId parameter
+Add a user with the userIds parameter to a SharePoint group with the groupId parameter.
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userIds 5
 ```
 
-Add multiple users with the userIds parameter to a SharePoint group with the groupId parameter
+Add multiple users with the userIds parameter to a SharePoint group with the groupId parameter.
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userIds "5,12"
 ```
 
-Add multiple users with the aadUserId parameter to a SharePoint group with the groupId parameter
```

Comment:
Should these be deleted or an option renamed!

`aadGroupIds` and `aadGroupNames` I mean..

Reply:
Yes, renamed options :)